### PR TITLE
Enable ppc64le Travis CI

### PR DESCRIPTION
**Pull request by nitkon**
Signed-off-by: Nitesh Konkar niteshkonkar@in.ibm.com

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by nitkon** [comment]
Travis for linux and linux-ppc64le is failing with:
```
go build -ldflags '-s -w -X main.gitCommit="4e3e1b04cbfab5864bbdb1c7208fdac890729609" -X main.buildInfo=1560317441' -tags "containers_image_ostree_stub     libdm_no_deferred_remove  seccomp selinux" -o bin/crio github.com/cri-o/cri-o/cmd/crio
# github.com/cri-o/cri-o/oci
oci/runtime_vm.go:175:20: undefined: strings.ReplaceAll
Makefile:120: recipe for target 'bin/crio' failed
make: *** [bin/crio] Error 2
The command "make" exited with 2.
```

**Comment by saschagrunert** [comment]
> Travis for linux and linux-ppc64le is failing with:
> 
> ```
> go build -ldflags '-s -w -X main.gitCommit="4e3e1b04cbfab5864bbdb1c7208fdac890729609" -X main.buildInfo=1560317441' -tags "containers_image_ostree_stub     libdm_no_deferred_remove  seccomp selinux" -o bin/crio github.com/cri-o/cri-o/cmd/crio
> # github.com/cri-o/cri-o/oci
> oci/runtime_vm.go:175:20: undefined: strings.ReplaceAll
> Makefile:120: recipe for target 'bin/crio' failed
> make: *** [bin/crio] Error 2
> The command "make" exited with 2.
> ```

This is because of the go version. This function only exists from go > 1.12.

**Comment by nitkon** [comment]
@saschagrunert : Yes, when I built crio locally I had to upgrade my go version > 1.12

**Comment by haircommander** [comment]
Neat
/lgtm
@saschagrunert I'll let you do the approval if you're okay with using more travis for now

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +26,10 @@ env:
     - CONTAINER_RUNTIME=docker
     - JOBS=4
 
+os:
+- linux
```
Comment (saschagrunert) [inline .travis.yml]: I guess this adds an additional target which we already have. If I look here:
https://travis-ci.com/cri-o/cri-o/builds/115294469

I see that we doubled the standard linux runs now with the one: `build, unit test and code coverage report`

File: .travis.yml [hunk 2]
```diff
@@ -23,6 +26,10 @@ env:
...
+- linux-ppc64le
```
Comment (saschagrunert) [inline .travis.yml]: Can we change the order of the jobs to first run the lints and then the tests like before?

**Comment by saschagrunert** [comment]
Maybe we can follow up on the mentioned issues in a separate PR? :)

**Pull request merged**